CreatubblesAPIClient/Sources/Database/DatabaseService.swift

```diff
@@ -30,48 +30,57 @@ class DatabaseService: NSObject
 {
     let realm = DatabaseService.prepareRealm()
     
-    fileprivate class func prepareRealm() -> Realm
+    fileprivate class func prepareRealmConfig() -> Realm.Configuration
     {
-        do
+        // The app group may be not accessible for testing purposes. That's why we added a failover below.
+        guard let appGroupURL: URL = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: AppGroupConfigurator.identifier)
+            else
+        {
+            return Realm.Configuration.defaultConfiguration
+        }
+        
+        let realmURL = appGroupURL.appendingPathComponent("db.realm")
+        var configuration = Realm.Configuration.defaultConfiguration
+        let originalDefaultRealmURL = configuration.fileURL
+        configuration.fileURL = realmURL
+        
+        if let defaultURL = originalDefaultRealmURL, FileManager.default.fileExists(atPath: defaultURL.path) && !FileManager.default.fileExists(atPath: realmURL.path)
         {
-            // The app group may be not accessible for testing purposes. That's why we added a failover below.
-            guard let appGroupURL: URL = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: AppGroupConfigurator.identifier) else {
-                return try Realm()
+            do
+            {
+                try FileManager.default.moveItem(atPath: originalDefaultRealmURL!.path, toPath: realmURL.path)
```

Comment:
Can we use `defaultURL` instead? It already has the `originalDefaultRealmURL` assigned and there's no need to force-unwrap.